bitnami/prometheus-operator/templates/exporters/kube-proxy/endpoints.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and .Values.kubeProxy.enabled .Values.kubeProxy.endpoints }}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: {{ template "prometheus-operator.fullname" . }}-kube-proxy
+  namespace: {{ .Values.kubeProxy.namespace }}
+  labels: {{- include "prometheus-operator.labels" . | nindent 4 }}
+    app.kubernetes.io/component: kube-proxy
+    k8s-app: kube-proxy
```

Comment:
Why do you need `k8s-app` labels? we already have  `app.kubernetes.io/name`, could they overlap?

Reply:
Good question. I think I put it in because in the Kubernetes documentation, I see that it's used for system critical things (https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#writing-a-daemonset-spec). But honestly, I think those docs were written before they formalized the labeling spec.

Anyway, I'll remove it, I don't think it's really doing anything.

Reply:
@dani8art I resolved the extra labels on the same commit. Note that the `selector` lines are **still** needed since the `kube-proxy` process is often labeled with `k8s-app: kube-proxy`, and so that selector needs to remain in place.

Reply:
Ok, sound good!